src/components/InlineSearchForm/index.tsx

```diff
@@ -0,0 +1,54 @@
+/** renders a form inline, containing search field and search button */
+import { Field, Form, Formik } from 'formik';
+import React from 'react';
+import { Button, FormGroup } from 'reactstrap';
+import { SEARCH } from '../../constants';
+
+/** Interface describing the form fields data */
+export interface FieldProps {
+  searchText: string;
+}
+
+/** default/initial values for the form fields */
+const defaultInitialValues: FieldProps = {
+  searchText: '',
+};
+
+export interface Props {
+  handleSubmit?: (values: FieldProps) => void;
+  inputId: string;
+  inputPlaceholder: string;
+}
+
+const defaultProps: Props = {
+  inputId: 'search',
+  inputPlaceholder: 'Search',
+};
+
+const InlineSearchForm = (props: Props) => {
```

Comment:
@p-netm do you think we should add more docstrings to this module, for consistency?

Reply:
will do